css-ecommerce-page/styles.css

```diff
@@ -0,0 +1,75 @@
+* {
+  font-family: "DM Sans", sans-serif;
+}
+
+.container {
+  position: relative;
+  width: 100%;
+  margin: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.center-align {
+  align-content: center;
+}
+
+.align-end {
+  align-items: flex-end;
+}
+
+.justify-evenly {
+  justify-content: space-evenly;
+}
+
+.text-align {
+  text-align: center;
+}
+
+.content {
+  margin: auto 3em;
+}
+
+.column {
+  flex-direction: column;
+  text-align: center;
+  width: 100%;
```

Comment:
the width of the column class should match its intended amount of horizontal space, in this case, that would be 20%.